src/influx.go

```diff
@@ -37,56 +38,50 @@ func newInflux(url, db, user, pass string) *Influx {
 }
 
 func (i *Influx) Check(retry int) bool {
-	resp_time, _, err := i.cli.Ping(i.timeout)
+	respTime, _, err := i.client.Ping(i.timeout)
+	for index := 1; index <= retry && err != nil; index++ {
+		log.Warn("Influx disconnected. Reconnecting ", index+1, " of ", retry, "...")
+		time.Sleep(time.Duration(1) * time.Second)
+		respTime, _, err = i.client.Ping(i.timeout)
+	}
+
```

Comment:
Is it connecting is a different go routine?
Asking because the original code was calling `Connect` explicitly